charts/pact-broker/values.yaml

```diff
@@ -504,3 +504,13 @@ serviceAccount:
 
   # -- Name of image pull secrets that should be attached to the service account
   imagePullSecrets: []
+
+# -- Additional containers to add to the Pact Broker pods
+extraContainers: []
```

Comment:
Apologies for late reply, project business :) 

I would put this in the `broker.` object as it's related to the addition of extra containers to the broker deployment.